hazelcast/src/main/java/com/hazelcast/datastore/JdbcDataStoreFactory.java

```diff
@@ -35,23 +35,33 @@
  */
 @Beta
 public class JdbcDataStoreFactory implements ExternalDataStoreFactory<DataSource> {
```

Comment:
not directly in the scope of this PR, but noticed that having a javaDoc to tell whether implementation expects any  thread-safety guarantee would be helpful.